src/main/java/com/mitrais/chipper/temankondangan/backendapps/service/impl/EventServiceImpl.java

```diff
@@ -95,30 +97,28 @@ public Event create(Long userId, CreateEventWrapper wrapper) {
 	@Override
 	public Event edit(Long userId, EditEventWrapper wrapper) {
 		Event event = eventRepository.findById(wrapper.getEventId())
-				.orElseThrow(() -> new ResponseStatusException(HttpStatus.BAD_REQUEST, "Error: Event not found!"));
+				.orElseThrow(() -> new ResourceNotFoundException("Event", "id", wrapper.getEventId()));
 
-		if(!event.getUser().getUserId().equals(userId)) {
-			throw new ResponseStatusException(HttpStatus.UNAUTHORIZED, "Error: Users are not authorized to edit this event");
+		if (!event.getUser().getUserId().equals(userId)) {
+			throw new ResponseStatusException(HttpStatus.UNAUTHORIZED,
```

Comment:
IDK, If we throw ResponseStatusException, does the exception handler in controller catch it? Have u test it?

Reply:
it should be, I have tried it with another ResponseStatusException but different HttpStatus, but again, the HttpStatus shouldn't be concerned

Reply:
Oo okay, got it